ivy/functional/backends/tensorflow/__init__.py

```diff
@@ -148,6 +150,45 @@
 
 native_inplace_support = False
 
+backend = "tensorflow"
+
+
+def _handle_inplace_mode():
+    if not hasattr(ivy, "inplace_mode") or ivy.inplace_mode not in [
+        "lenient",
+        "strict",
+    ]:
+        ivy.set_inplace_mode("lenient")
+
+    if ivy.inplace_mode == "lenient":
+        message = (
```

Comment:
I think we can just throw a warning here if the `inplace_mode` is `lenient`, given that we're anyway throwing an exception if it was `strict`, and given that `strict` isn't the default, we can assume the user knows that to be the case. 
Also, lets probably shorten this a bit to,
```
The current backend: '{backend}' does not support inplace updates natively. 
Ivy would quietly create new arrays when using inplace updates with this 
backend, leading to memory overhead (same applies for views). If you want to 
control your memory management, consider doing ivy.set_inplace_mode('strict') 
which should raise an error whenever an inplace update is attempted with this 
backend. 
```

Reply:
But this is the first time where the user encounter these warnings, so shouldn't we give him all information about inplace modes ?